### PR TITLE
[15.0][MIG] account_menu: migrate module from 14.0 to 15.0

### PR DESCRIPTION
**Pull request by AlvaroTForgeFlow**

### CODE REVIEW

**Comment by AlvaroTForgeFlow** [comment]
@ForgeFlow

File: account_menu/readme/ROADMAP.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* Suggest to rename to something like `account_usability` in 15.0, given that
```
Comment (ChrisOForgeFlow) [inline account_menu/readme/ROADMAP.rst]: I also agree with this recommendation

**Review by ChrisOForgeFlow — CHANGES_REQUESTED** [review]
Please let us evaluate the feasibility of renaming, since the module has several additional features to only the menus.

File: account_menu/readme/ROADMAP.rst (hunk 1 above)
Comment (AlvaroTForgeFlow) [reply account_menu/readme/ROADMAP.rst]: Understood. I don't think a renaming is needed since menus are still the main functionality, right?

Comment (ChrisOForgeFlow) [reply account_menu/readme/ROADMAP.rst]: Okay, they are the menu feature, however the module has more features than the menu, so the name change could give more sense to the rest of the added features, what do you think?

Comment (AlvaroTForgeFlow) [reply account_menu/readme/ROADMAP.rst]: I see your point. I'll make sure to make those changes asap.

Comment (AlvaroTForgeFlow) [reply account_menu/readme/ROADMAP.rst]: @ChrisOForgeFlow It should be good now.

**Review by ChrisOForgeFlow — APPROVED** [review]
+1

**Review by etobella — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
Functional review

**Review by HaraldPanten — DISMISSED** [review]
Please, check pre-commit

**Comment by AlvaroTForgeFlow** [comment]
Oddly enough, I tried to re-run it in local and the pre-commit passed succesfully.

**Comment by GuillemCForgeFlow** [comment]
@HaraldPanten It seems like the runboat is failing because of this error when building: 
`× python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error in vatnumber setup command: use_2to3 is invalid.
      [end of output]`
I've seen other PR's of this repo which seem to have the same error. Any idea on how should we fix this?

**Comment by HaraldPanten** [comment]
Hhhhmmm... maybe @sbidoul or @pedrobaeza could shed some light here.

**Comment by sbidoul** [comment]
@GuillemCForgeFlow @HaraldPanten the problem here is that `base_vat_optional_vies` depends on `vatnumber`, which is unmaintained and incompatible with the latest `setuptools`. 

image


Odoo itself has removed the dependency on `vatnumber`. I think we should do the same, as we don't want to pin an old `setuptools` version in the `oca/oca-ci` image just for this.

**Comment by rafaelbn** [comment]
We have the same problem about `vatnumber `in https://github.com/OCA/account-financial-tools/pull/1360

**Comment by rafaelbn** [comment]
Hello @HaraldPanten , please could you re-review? 😄 this is ready!

**Comment by rafaelbn** [comment]
@AaronHForgeFlow please could you review this PR from @Forgeflow ? thanks!

**Comment by rafaelbn** [comment]
Solved

**Review by rafaelbn — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
LGTM

**Pull request merged**